scripts/test-setup.sh

```diff
@@ -0,0 +1,27 @@
+trap 'for killable in $(jobs -p); do kill -9 $killable; done' EXIT
```

Comment:
The idea of this trap was to kill leftover devnet processes; is it still applicable? should it be moved?

Reply:
devnet script is still used when not running via `dev-test`